packages/SwingSet/src/kernel/messageResult.js

```diff
@@ -0,0 +1,68 @@
+/* global harden */
+
+import { assert, details } from '@agoric/assert';
+import { insistCapData } from '../capdata';
+
+export function makeMessageResult(message, resultPolicy, panic) {
+  let currentStatus = 'pending';
+  let eventualResolution;
+
+  assert(
+    resultPolicy === 'logAlways' ||
+      resultPolicy === 'logFailure' ||
+      resultPolicy === 'panic' ||
+      resultPolicy === 'ignore',
+    details`invalid result policy ${resultPolicy}`,
+  );
+  assert.typeof(panic, 'function');
+
+  function log() {
+    // prettier-ignore
+    console.log(
```

Comment:
You might make this `console.error` rather than `console.log`, I believe `.log` is not displayed in some of our runtimes (in particular I think `agoric-cli` runs its child processes with `stdout` routed to `/dev/null`, and only `stderr` makes it through). It is, of course, an arms race between log-producers and clutter-reducers, but hopefully we can address that with the ever-distant "grand logging unification" scheme that we'll get to some day.

Reply:
Changed to `console.error`